### DISCUSSION TITLE
Challenges

### DISCUSSION THREAD

**Discussion by ghost** [Cyfrin/foundry-full-course-cu#2940] [General]
It has been particularly challenging for to become a blockchain developer. I am a beginner and it has been scary, I am seeking help.

**Comment by EngrPips** [comment]
Hello @yush48, Welcome aboard and the whole community is always available to help you whenever you get stuck, so don't be scared and make sure you do everything within your power to be successful.

**Comment by heheboii11** [comment]
Hey @yush48, we are here for you and ready to help ASAP. Don't worry and go ahead with the course. It really can make you a good developer!!!

**Reply by EngrPips** [reply]
Indeed
